### ISSUE TITLE
Wiki opens as a blank page in TW In the Sky

### ISSUE THREAD

**Issue by af4jm** [twcloud/tw5-dropbox#14]
I've got 4 TiddlyWikis that I use regularly, living in DropBox. They all work in TiddlyDesktop and opening the file directly in the browser. They all used to work in TW in the Sky as well, but now one of them opens as a plain blank page, while the others continue to work normally. I see the same behavior on Android & Windows, in Chrome & Edge (yes, both based on Chromium).

This has the effect, for me, of removing mobile access to that wiki. Ironically it's the wiki I reference most often on mobile (or did, before this issue started).

I've tried running the problem wiki through the upgrader (which in the past has fixed a problem---unrelated to TW in the Sky---from removing a problematic extension), but that had no effect.

I'm open for suggestions. Thanks!

**Comment by Arlen22** [comment]
Sorry, this is a really simple project as typescript projects go, but I don't really maintain this currently.

**Comment by af4jm** [comment]
fair enough... i was able to import all my tiddlers into a fresh TW, which did resolve the issue... if it happens again I might try to investigate further

**Comment by wright-david** [comment]
Just used the upgrade wizard from TiddlyWiki 5.1.17 to 5.1.23 on a Dropbox hosted TW file, and had the blank page problem too.

The upgraded TiddlyWiki opened OK from TiddlyDesktop v0.0.14.

Successfully used the workaround above of starting with an empty 5.1.23 TiddlyWiki and importing from the 5.1.17 file. 
_Thanks @af4jm_ 

FWIW the file was 3.4Mb (5.1.17) -> 3.8Mb (5.1.23)

**Comment by Vhou-Atroph** [comment]
One potential cause (but almost certainly not the only one) of this issue seems to be the usage of certain specific emojis in tiddlers such as 👍 and 📣. I'm not certain, but it may have something to do with the `openFile` function in twcloud.ts. This may be related to https://github.com/microsoft/TypeScript/issues/49150 in some way, but I don't use TypeScript myself so I cannot say for sure. 

Credit to @roxje for discovering this
